packages/paste-website/src/pages/components/paragraph/index.mdx

```diff
@@ -0,0 +1,156 @@
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<LivePreview scope={{Paragraph}} language="jsx">
+  {`<Paragraph>
+  Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <em>Curabitur blandit tempus</em> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+</Paragraph>`}
+</LivePreview>
+
+## Guidelines
+
+By default, the paragraph component should provide consistent values for font-size, line-height, and margin-bottom across themes.
+
+### About Paragraphs
+
+Paragraphs are distinct sections of content, typically dealing with a single theme, and are indicated by a new line. Paragraphs are often used for a majority of text on the page and should be able to support a variety of information-dense layouts.
+
+Paragraph is part of the Typography component, which includes Header and List as well.
+
+### Accessibility
+
+Paragraph text should be legible, and easy to read. It should also provide a clear amount of space beneath a block of text to visually separate itself from adjacent content.
+
+- Breaking content up into paragraphs helps screen readers and assistive technology provide shortcuts for users to skip between them
+- Paragraph text should meet AA requirements (4.5:1) for color contrast from itself and the background color
+
+## Examples
+
+### Default Paragraph
+
+<LivePreview scope={{Paragraph}} language="jsx">
+  {`<article>
+  <Paragraph>
+    Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <em>Curabitur blandit tempus</em> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+  </Paragraph>
+  <Paragraph>
+    Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <strong>Curabitur blandit tempus</strong> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+  </Paragraph>
+</article>`}
+</LivePreview>
+
+## Composition Notes
+
+### When to Use a Paragraph
+
+Paragraphs should be used for most text blocks. Paragraphs provide defaults for font-size, line-height, and margin-bottom. Paragraphs can be used within other components as well. Typically, these are anywhere from 1-5 sentences. Use paragraphs like you would when authoring a word document, to group sentences of a similar theme.
+
+<DoDont>
+  <Do title="Do" body="Use paragraphs for text blocks." center />
```

Comment:
I really think we should go the extra mile here and prevent merging docs without examples.  It should be pretty straightforward to come up with code examples for these.